src/components/ProjectDocumentsFolders/index.js

```diff
@@ -0,0 +1,268 @@
+import styled from '@emotion/styled/macro';
+import moment from 'moment';
+import React, {useCallback, useState} from 'react';
+import FileIcon, {defaultStyles} from 'react-file-icon';
+
+import fbt from '../../fbt/fbt.macro';
+import {useMutation, useQuery} from '../../utils/apollo-hooks';
+import {documentTypes} from '../../utils/constants';
+import {formatName} from '../../utils/functions';
+import {ReactComponent as SectionIcon} from '../../utils/icons/section-icon.svg';
+import {REMOVE_ATTACHMENTS, UPLOAD_ATTACHMENTS} from '../../utils/mutations';
+import {
+	ActionCell,
+	Actions,
+	Cell,
+	HeaderCell,
+	primaryPurple,
+	primaryWhite,
+	Row,
+	RowHeader,
+	Table,
+	TaskIcon,
+} from '../../utils/new/design-system';
+import {
+	GET_PROJECT_DATA,
+	GET_PROJECT_DATA_WITH_TOKEN,
+} from '../../utils/queries';
+import IconButton from '../IconButton';
+import MaterialIcon from '../MaterialIcon';
+import UploadDashboardButton from '../UploadDashboardButton';
+
+const DocumentFolders = styled('div')`
+	flex: 1;
+`;
+
+const SectionHeaderDocument = styled('div')`
+	display: flex;
+	align-items: center;
+	cursor: pointer;
+`;
+
+const SectionContentDocument = styled('div')`
+	margin-left: 1.5rem;
+	margin-bottom: 1rem;
+	height: ${props => (props.open ? 'auto' : '0px')};
+	overflow: hidden;
+`;
+
+const TaskHeaderDocument = styled('div')`
+	display: flex;
+	align-items: center;
+	cursor: pointer;
+`;
+
+const TaskContentDocument = styled('div')`
+	margin-left: 4.4rem;
+	height: ${props => (props.open ? 'auto' : '0px')};
+	overflow: hidden;
+`;
+
+const DocTypeList = styled('div')`
+	display: flex;
+	margin-bottom: 1rem;
+`;
+
+const DocType = styled('div')`
+	background: ${props => (props.active ? primaryPurple : 'transparent')};
+	color: ${props => (props.active ? primaryWhite : primaryPurple)};
+	margin-right: 1rem;
+	padding: 0.3rem 0.5rem;
+	border-radius: 5px;
+	border: solid 1px ${primaryPurple};
+	cursor: pointer;
+
+	&:hover {
+		background: ${primaryPurple};
+		color: ${primaryWhite};
+	}
+`;
+
+const UploadContainer = styled('div')`
+	margin-top: 1rem;
+`;
+
+const headerCells = [
+	{
+		label: 'Type',
+		prop: 'extension',
+	},
+	{
+		label: 'Fichier',
+		prop: 'filename',
+	},
+	{
+		label: 'Section',
+		prop: 'sectionName',
+	},
+	{
+		label: 'Tâche',
+		prop: 'itemName',
+	},
+	{
+		label: 'Date',
+		prop: 'datetime',
+	},
+	{
+		label: 'Uploadeur',
+		prop: 'ownerName',
+	},
+];
+
+const ProjectDocumentsFolders = ({projectId, customerToken}) => {
+	const [active, setActive] = useState(documentTypes[0].type);
+	const [uploadAttachments] = useMutation(UPLOAD_ATTACHMENTS);
+	const [removeFile] = useMutation(REMOVE_ATTACHMENTS);
+	const {data: projectData, error} = useQuery(
+		customerToken ? GET_PROJECT_DATA_WITH_TOKEN : GET_PROJECT_DATA,
+		{
+			variables: {projectId, token: customerToken},
+			suspend: true,
+		},
+	);
+	const [sorting, setSorting] = useState('Date');
```

Comment:
since it compares to prop, should be that instead

```suggestion
	const [sorting, setSorting] = useState('datetime');
```